BowlingBall.Tests/GameFixture.cs

```diff
@@ -1,14 +1,108 @@
 using System;
 using Xunit;
-
 namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        Game game=null;

```

Comment:
Make game a private variable as you don't need this instance outside Fixture class. Follow naming convention while making it private